redbeams/src/main/java/com/sequenceiq/redbeams/converter/spi/DBStackToDatabaseStackConverter.java

```diff
@@ -66,6 +68,29 @@ private DatabaseServer buildDatabaseServer(DBStack dbStack) {
             return null;
         }
 
+        Map<String, Object> params = buildParameters(dbStack);
+        SecurityGroup securityGroup = dbStackDatabaseServer.getSecurityGroup();
+
+        DatabaseServer.Builder builder = DatabaseServer.builder()
+                .serverId(dbStackDatabaseServer.getName())
+                .flavor(dbStackDatabaseServer.getInstanceType())
+                .engine(getDatabaseEngine(dbStackDatabaseServer))
+                .connectionDriver(dbStackDatabaseServer.getConnectionDriver())
+                .rootUserName(dbStackDatabaseServer.getRootUserName())
+                .rootPassword(dbStackDatabaseServer.getRootPassword())
+                .port(dbStackDatabaseServer.getPort())
+                .useSslEnforcement(determineSslEnforcement(dbStack))
+                .storageSize(dbStackDatabaseServer.getStorageSize())
+                .security(securityGroup == null ? null : new Security(Collections.emptyList(), securityGroup.getSecurityGroupIds()))
+                // TODO / FIXME converter caller decides this?
```

Comment:
do we need this todo?

Reply:
It was already there added by the original author. I did not know if the status ever needed to be fixed so left it as is. But can be indeed removed if you say so. 😄

Reply:
ah :D ok leave there.